test/commons/color/get-foreground-color.js

```diff
@@ -14,8 +14,8 @@ describe('color.getForegroundColor', function() {
 
 	it('should return the blended color if it has alpha set', function() {
 		fixture.innerHTML =
-			'<div style="height: 40px; width: 30px; background-color: #800000;">' +
-			'<div id="target" style="height: 20px; width: 15px; color: rgba(0, 0, 128, 0.5);' +
+			'<div style="height: 40px; background-color: #800000;">' +
```

Comment:
I had to update these tests as my code changes made them more accurate than before. Before my changes, we would test only the first line of this fixture. However, if we look at the result of the fixture, we can see that there should be 3 background colors and 3 foreground colors, but we only calculate 1.

![image](https://user-images.githubusercontent.com/2433219/73388962-6d052900-4290-11ea-8254-11b95a96e266.png)

So I updated the tests to just focus on the single color with alpha/opacity.